package.json

```diff
@@ -21,6 +21,7 @@
     "bootstrap": "3.3.6",
     "bootstrap-slider": "7.1.0",
     "closure-util": "1.15.1",
+    "moment-timezone": "^0.5.9",
```

Comment:
Ok. This is how `Npm install --save `adds package version by default. That led me to look at other packages -  `snyk `and `angular-moment` are also set like that, When I looked at `angular-moment` release history, we are few versions behind (beta5 added bugfix of `amTimeAgo` we want to work with now + we use latest `moment `and `moment-timezone` packages which look like they not supported by currently used older version of `angular-moment`?)
https://github.com/urish/angular-moment/releases